src/hypergraph/viz/assets/constraint-layout.js

```
@@ -703,6 +767,65 @@
     return separationConstraints;
   };
 
+  const alignDataNodesToSources = (rows, edges, layoutConfig) => {
+    if (!DATA_NODE_ALIGN_WEIGHT) return;
+    const alignWeight = Math.max(0, Math.min(1, DATA_NODE_ALIGN_WEIGHT));
+    const coordPrimary = layoutConfig.coordPrimary;
+    const nodeById = new Map();
+    rows.forEach((row) => row.forEach((node) => nodeById.set(node.id, node)));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)